pkg/controller/statefulset/stateful_set.go

```diff
@@ -141,22 +141,22 @@ func NewStatefulSetController(
 }
```

Comment:
a few more in:
pkg/controller/statefulset/stateful_set_status_updater.go
pkg/controller/statefulset/stateful_pod_control.go